### DISCUSSION TITLE
【求助】MHYYYY怎么了？

### DISCUSSION THREAD

**Discussion by guozichang2007** [share121/inter-knot#2361] [其他]
这个人已经二十多天没上崩铁号了，版本更新都不玩的吗？
![v2-7915e491d99f8d4fd171043dc9974790_xll](https://github.com/user-attachments/assets/c8eb4932-715e-4bd8-a6ad-8018219c25a0)

**Comment by Lixia256** [comment]
似乎是因为在璃月试图释放禁忌知识被钟离先生天动万象了。

**Comment by Qweulin** [comment]
不开玩笑，这人不是已经进去踩缝纫机了吗<img src="https://img-static.mihoyo.com/communityweb/upload/6b72e22e82737bda6bccf29692bee918.png" alt="缉拿归案" width="75" >
